Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install python3-pip python3 python-is-python3 -y
+RUN pip install --upgrade eth-ape
+RUN ape plugins add ens -y
```

Comment:
For scripts, it is generally more readable to use the longer option names, such as `--yes` in this case instead of `-y`. Shorthand options are great for adhoc, long-hand options are great for scripts.

Reply:
I'll make the requested changes for this. I also I got an email with your note about docker compose, not sure where that comment went, but would love to chat with you about this, maybe at Wednesday meeting.

Reply:
Ah yeah, the docker-compose thing might be more complicated than I though. It might make sense for ape plugin dependencies though, like running the hardhat network.

Another usability for `ape` would be to copy other package managers and allow multiple args for the plugins when adding. That way, you can just do:

```
ape plugins add ens etherscan infura solidity 
```

and it will install all 4.

Reply:
One thing we can do is create a `build_ape.sh` script that takes parameters such as which plugins to use. Or maybe it figures it out other ways. Either way, this script can be copied in the container and ran with parameters set from environment variables e.g. `--build-arg ARG_NAME=ARG_VALUE`

Reply:
Yes I believe the reason we went with dockerfile and not compose is that we didn't want to have it spinning up an app, we wanted it to be a CLI command.

Reply:
https://github.com/ApeWorX/ape/issues/383